pkgs/development/tools/argc/default.nix

```diff
@@ -6,16 +6,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "argc";
-  version = "1.3.0";
+  version = "1.4.0";
 
   src = fetchFromGitHub {
     owner = "sigoden";
     repo = pname;
```

Comment:
I don't think it matters enough that I would actively change this in existing files, but I've switched to just using string literals for new things I've packaged (also since I've been using nix-init exclusively)